crm_salesperson_planner/models/crm_lead.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Sygel - Valentin Vinagre
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = "crm.lead"
+
+    crm_salesperson_planner_visit_ids = fields.Many2many(
+        comodel_name="crm.salesperson.planner.visit",
+        relation="crm_salesperson_planner_visit_crm_lead_rel",
+        string="Visits",
+        copy=False,
+        domain="[('partner_id', 'child_of', partner_id)]",
```

Comment:
Isn't this domain going to hide some legit linked visits if changing the partner for whatever reason?

Reply:
I'm not sure about that, but if you have investigated it, you tell me.

Reply:
If some of a partner's contacts change parent, it does not affect the relationship already created between the visit and the opportunity.